wagtail/images/static_src/wagtailimages/scss/add-multiple.scss

```diff
@@ -94,7 +94,7 @@
         width: 1em;
         font-size: 10em;
         line-height: 1.4em;
-        color: lighten($color-grey-4, 4%);
+        color: hsl(css-lighten(get-color('grey-4'), 4%)...);
```

Comment:
I’ve never seen this syntax before, can you explain what’s going on here?

Reply:
Assuming you mean the `...`, that's spreading a list of 3 values onto the 3 arguments of sass's hsl function. Can't find the sass documentation but it seems to work the same as [the JS equivalent](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax). More detailed explanation below in case that's not what you meant.

`get-color` (defined [here](https://github.com/JNaftali/wagtail/blob/e17455a4866f663f76be4f30eb5247d604e389e6/client/scss/settings/_variables.scss#L113-L115)) returns a list of 3 css variables (in this case `var(--color-grey-4-hue)` followed by the equivalent for saturation and lightness)
`css-lighten` (defined [here](https://github.com/JNaftali/wagtail/blob/e17455a4866f663f76be4f30eb5247d604e389e6/client/scss/settings/_variables.scss#L60-L65)) takes that list and returns a new list, replacing the last element (`var(--color-grey-4-lightness)`) with a calc statement that lightens it (`calc( var(--grey-4-lightness) - 4% )`). In effect the same as the sass lighten function that it's replacing, except the calculation happens at runtime (in the browser) not at build time.
`hsl` is the sass hsl function. It takes 3 arguments, so the `...` spreads the list of 3 values returned by `css-lighten` out onto those 3 arguments.

Reply:
Thank you, I had never seen `...` and can’t even find documentation about it on the Sass website.